.travis.yml

```diff
@@ -28,7 +28,8 @@ env:
     - SETUP_CMD="${PYTEST_FLAGS}"
     - BUILD_CMD="pip install -e package/ && (cd testsuite/ && python setup.py build)"
     - CONDA_MIN_DEPENDENCIES="mmtf-python mock six biopython networkx cython matplotlib scipy griddataformats hypothesis gsd codecov"
-    - CONDA_DEPENDENCIES="${CONDA_MIN_DEPENDENCIES} seaborn>=0.7.0 clustalw=2.1 netcdf4 scikit-learn joblib>=0.12"
+    # Temporarily cap seaborn to 0.9 since 0.10 drops Python 2 support
```

Comment:
Can you add a link to the actual issue so that, once the issue is resolved, we can remove the pin?